### PR TITLE
Removes `loader` folder from the root

### PR DESCRIPTION
**Pull request by twokul**

### CODE REVIEW

File: lib/concatenate-es6-modules.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ var defaultOptions = {
 */
 module.exports = function concatenateES6Modules(inputTrees, options) {
   var mergedOptions = merge(defaultOptions, options || {});
-  var loader        = vendoredPackages['loader'];
+  debugger
```
Comment (rwjblue) [inline lib/concatenate-es6-modules.js]: ?

**Comment by rwjblue** [comment]
:+!:

**Comment by twokul** [comment]
build failed, looking into it

**Comment by twokul** [comment]
@rwjblue good to go

**Pull request merged**